zschema/tests.py

```diff
@@ -17,10 +17,14 @@ class LeafUnitTests(unittest.TestCase):
 
     def test_valid(self):
         for leaf in VALID_LEAVES:
+            if leaf.DEPRECATED:
+                continue
             leaf().validate(leaf.__name__, leaf.VALID)
 
     def test_invalid(self):
         for leaf in VALID_LEAVES:
+            if leaf.DEPRECATED:
+                continue
```

Comment:
Why are we just excluding deprecated leaves from tests? It seems to me that either they should be removed, or they should continue to be tested with the rest of the codebase.